app/services/authn_context_resolver.rb

```diff
@@ -54,18 +77,47 @@ def no_identity_proofing_vot
     end
   end
 
+  def acr_results
+    decorate_acr_result_with_user_context(
+      acr_result_with_sp_defaults,
+    )
+  end
+
   def acr_result_with_sp_defaults
+    decorate_acr_result_with_sp_defaults(
+      acr_result_without_sp_defaults,
+    )
+  end
+
+  # @param [Vot::Parser::Result] result Parser result to decorate.
+  # @return [Vot::Parser::Result]
+  def decorate_acr_result_with_sp_defaults(result)
     result_with_sp_aal_defaults(
       result_with_sp_ial_defaults(
-        acr_result_without_sp_defaults,
+        result,
       ),
     )
   end
 
+  # @param [Vot::Parser::Result] result Parser result to decorate.
+  # @return [Vot::Parser::Result]
+  def decorate_acr_result_with_user_context(result)
```

Comment:
this logic is close but not quite right yet -- we need to return `biometric_comparison: true` if the bio?=required is passed in.